.gitignore

```diff
@@ -1,20 +1,132 @@
-# Dependencies
-/node_modules
+# This file was copied from <https://github.com/github/gitignore/blob/main/Node.gitignore>.
 
-# Production
-/build
```

Comment:
I am pretty sure this is a Docusaurus specific directory. We would not want static builds commited after local testing.